### PR TITLE
Add DetailedReport

### PR DESCRIPTION
**Pull request by popstas**
I've some corrected @therealkevinard's code in his develop branch and added some undocumented fields: project_color and project_hex_color

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -124,6 +124,25 @@ type TimeEntry struct {
...
+type DetailedTimeEntry struct {
+	ID              int        `json:"id"`
+	Pid             int        `json:"pid"`
+	Tid             int        `json:"tid"`
+	Uid             int        `json:"uid"`
+	User            string     `json:"user,omitempty"`
+	Description     string     `json:"description"`
+	Project         string     `json:"project"`
+	ProjectColor    string     `json:"project_color"`
+	ProjectHexColor string     `json:"project_hex_color"`
```
Comment (jason0x43) [inline main.go:139]: Where do these color entries come from? They're not described in the [API docs](https://github.com/toggl/toggl_api_docs/blob/master/reports/detailed.md).

Comment (therealkevinard) [reply main.go:139]: I found those documented in the summary report (ref: https://github.com/toggl/toggl_api_docs/blob/master/reports/summary.md#project-colors) 

Sometimes, Toggl seems like one of those apis whose code moves quicker than its docs

Comment (jason0x43) [reply main.go:139]: Heh, it looks like the project colors have actually been around for a while and I'd just never noticed them (https://github.com/toggl/toggl_api_docs/commit/3c5be305a23e14f209d8879471e7ffce21e3fe49). Well, we might as well add the color fields to the Summary entries, too. They get `color` and `hex_color` strings.

Comment (popstas) [reply main.go:139]: Colors not described in docs, but present in response, I am using it in my project. Agreed, we can add it to Summary report too, should we do it in this PR or outside?

Comment (jason0x43) [reply main.go:139]: Sorry for taking so long to get back to you. Yeah, add the colors to summary report and we'll call it good.

**Comment by popstas** [comment]
Sorry, I'm send PR from master branch, but I made some more commits in it, therefore I'm created new branch and added colors to summary report - #8, I'm closing this PR.

**Pull request closed without merging**